pkg/packages.builds

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
```

Comment:
This was the simplest solution since `pkg\projects\dir.props` and `pkg\projects\dir.traversal.targets` aren't intended to apply to this file and @eerhardt wasn't happy with special casing the imports to `..\dir.blah`.  We don't have to deal with this in other repos where packages.builds is also not a part of the same folder layout which handles buildrids / filtering.